mobile/voice_journal_app/lib/home.dart

```diff
@@ -1,43 +1,72 @@
 import 'package:flutter/material.dart';
-
+import 'RecordingPage.dart'; //Importing recording page
```

Comment:
We seem to be merging changes from other PRs here, this is generally bad practice. I know we need to update this page to add the navigation to the recording page, but let's hold off on it until Anika's home page changes get merged. Sorry if I made it confusing!